bigchaindb/web/websocket_server.py

```diff
@@ -50,13 +50,13 @@ def _multiprocessing_to_asyncio(in_queue, out_queue, loop):
 
 def eventify_block(block):
     for tx in block['transactions']:
-        try:
-            asset_id = tx['asset']['id']
-        except KeyError:
-            asset_id = tx['id']
+        if tx.asset:
+            asset_id = tx.asset.get('id', tx.id)
+        else:
+            asset_id = tx.id
         yield {'height': block['height'],
                'asset_id': asset_id,
-               'transaction_id': tx['id']}
+               'transaction_id': tx.id}
```

Comment:
While we're busy fixing the events API, we may as well fix the docs about it. If you look in the docs subsection

http://docs.bigchaindb.com/projects/server/en/master/events/websocket-event-stream-api.html#id3

it gives this "Example message:"

```js
{
    "transaction_id": "<sha3-256 hash>",
    "asset_id": "<sha3-256 hash>",
    "block_id": "<int>"
}
```

Clearly, `'block_id'` should be replaced by `'height'`.

Reply:
The sentence right before that also refers to the "containing block’s ID." but it should say "containing block's height."

Reply:
I'll use `int`. Given that the biggest integer in JavaScript is `9007199254740991`, and if a BigchainDB network creates a block every second, we will have problems in 286 million years.

At that point in time we would have other problems to deal with, since _all the continents on Earth may fuse into a supercontinent_ (from [Wikipedia](https://en.wikipedia.org/wiki/Timeline_of_the_far_future)).